elsa-typed-builtin.el

```diff
@@ -350,6 +350,67 @@
 (put 'delete-char 'elsa-type (elsa-make-type Int -> Mixed -> Nil))
 (put 'self-insert-command 'elsa-type (elsa-make-type Int -> Nil))
 
+;; File: process.c
+(put 'processp 'elsa-type (elsa-make-type Mixed -> Bool))
+(put 'get-process 'elsa-type (elsa-make-type String -> Process?))
+(put 'delete-process 'elsa-type (elsa-make-type (Process | Buffer | String | Nil) -> Nil))
+(put 'process-status 'elsa-type (elsa-make-type (Process | Buffer | String | Nil) -> Symbol?))
+(put 'process-exit-status 'elsa-type (elsa-make-type Process -> Int))
+(put 'process-id 'elsa-type (elsa-make-type Process -> Number?))
+(put 'process-name 'elsa-type (elsa-make-type Process -> String))
+(put 'process-command 'elsa-type (elsa-make-type Process -> List String | Bool))
+(put 'process-tty-name 'elsa-type (elsa-make-type Process -> String?))
+(put 'set-process-buffer 'elsa-type (elsa-make-type Process Buffer -> Buffer))
+(put 'process-buffer 'elsa-type (elsa-make-type Process -> Buffer))
+(put 'process-mark 'elsa-type (elsa-make-type Process -> Marker))
+(put 'set-process-filter 'elsa-type (elsa-make-type Process -> (Process -> String -> Nil) -> (Process -> String -> Nil)))
+(put 'process-filter 'elsa-type (elsa-make-type Process -> (Process -> String -> Nil)))
+(put 'set-process-sentinel 'elsa-type (elsa-make-type Process -> (Process -> String -> Nil) -> Nil))
+(put 'process-sentinel 'elsa-type (elsa-make-type Process -> (Process -> String -> Nil)))
+(put 'set-process-window-size 'elsa-type (elsa-make-type Process -> Int -> Int -> Bool))
+(put 'set-process-inherit-coding-system-flag 'elsa-type (elsa-make-type Process -> Bool -> Bool))
+(put 'set-process-query-on-exit-flag 'elsa-type (elsa-make-type Process -> Bool -> Bool))
+(put 'process-query-on-exit-flag 'elsa-type (elsa-make-type Process -> Bool))
+;; (put 'process-contact 'elsa-type (elsa-make-type))
+;; (put 'process-plist 'elsa-type (elsa-make-type))
+;; (put 'set-process-plist 'elsa-type (elsa-make-type))
+;; (put 'process-connection 'elsa-type (elsa-make-type))
+(put 'process-type 'elsa-type (elsa-make-type (Process -> Buffer -> String -> Nil) -> Symbol))
```

Comment:
This should be a union `|` not a function `->` right?

Reply:
indeed, thanks